sequoia-autocrypt "features" really just echo the features of the
sequoia-openpgp crate.

since features antagonize the ftp team, we'll just strip them all, and
have the autocrypt crate rely on the default features of
sequoia-openpgp.


--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,15 +28,7 @@
 
 [dependencies.sequoia-openpgp]
 version = "1"
-default-features = false
 
-[features]
-compression = ["sequoia-openpgp/compression"]
-compression-bzip2 = ["sequoia-openpgp/compression-bzip2"]
-compression-deflate = ["sequoia-openpgp/compression-deflate"]
-crypto-cng = ["sequoia-openpgp/crypto-cng"]
-crypto-nettle = ["sequoia-openpgp/crypto-nettle"]
-default = ["sequoia-openpgp/default"]
 [badges.gitlab]
 repository = "sequoia-pgp/sequoia"
 
